usage.go

```diff
@@ -8,20 +8,22 @@ var (
 	UsageConnect = `'alias' or 'transport://[host][/digi]/targetcall[?params...]'
 
 transport:
-  ardop:      ARDOP TNC
-  ax25:       AX.25 (Linux only)
-  telnet:     TCP/IP
-  serial-tnc: Serial AX.25 TNC
-  pactor:     SCS PTC modems
-  varahf:     VARA HF TNC
-  varafm:     VARA FM TNC
+  telnet:          TCP/IP
+  ardop:           ARDOP TNC
+  pactor:          SCS PTC modems
+  varahf:          VARA HF TNC
+  varafm:          VARA FM TNC
+  ax25:            AX.25 (Default - uses engine specified in config)
+  ax25+agwpe:      AX.25 (AGWPE/Direwolf)
+  ax25+linux:      AX.25 (Linux kernel)
```

Comment:
Might be worth calling this by name, `libax25`

Reply:
Do you mean in the comment or in the scheme? Or both maybe?

Although it is commonly associated with the kernel module, libax25 is actually the name of the C library providing utility functions for interface with the kernel's AX.25 module. It's not really a requirement to utilize the kernel's AX.25 feature and I was actually hoping to port the required functions to Go at some point, meaning we won't use libax25 at all.

But I see your point. Maybe `ax25+linux: AX.25 (Linux kernel aka "libax25")` could work? 🤔

Reply:
I suppose this is ok as-is. I always associated the kernel extension as `libax25` because that's the thing I needed to install to start using it, but it's accurate as you have it.